sklearn/utils/tests/test_testing.py

```diff
@@ -235,3 +238,129 @@ def f():
 
         if failed:
             raise AssertionError("wrong warning caught by assert_warn")
+
+
+# Tests for docstrings:
+
+def f_ok(a, b):
+    """Function f
+
+    Parameters
+    ----------
+    a : int
```

Comment:
Has this case been addressed? I think it's a very good validation check to add and test.